main.go

```diff
@@ -53,6 +55,8 @@ func main() {
 	fs.StringVar(&dbUser, "db-user", "postgres", "database username")
 	fs.StringVar(&dbPass, "db-pass", "", "database user password")
 	fs.StringVar(&dbURL, "database-url", "", "database connection URL")
+	fs.StringVar(&metricsTopic, "metrics-topic", "platform.upload.client-metrics", "topic on which to place metrics data")
+	fs.StringVar(&kafkaBootstrap, "kafka-bootstrap", "localhost:29092", "url of the kafka broker for the cluster")
```

Comment:
Let's default this to an empty string, and then conditionally create a Kafka producer if this value is non-zero.